using-timescaledb/backup.md

```diff
@@ -30,28 +43,34 @@ Restoring data from a backup currently requires some additional procedures,
 which need to be run from `psql`:
 ```sql
 CREATE DATABASE tutorial;
-ALTER DATABASE tutorial SET timescaledb.restoring='on';
+\c tutorial --connect to the db where we'll perform the restore
+CREATE EXTENSION timescaledb; 
+SELECT timescaledb_pre_restore();
 
 -- execute the restore (or from a shell)
 \! pg_restore -Fc -d tutorial tutorial.bak
 
--- connect to the restored db
-\c tutorial
-ALTER DATABASE tutorial SET timescaledb.restoring='off';
+
```

Comment:
Do you not have to `\c tutorial` again?

Reply:
Yes, but assuming you're doing it from inside of psql already is weird?

Reply:
nope you're already connected.